skymap_scanner/recos/__init__.py

```diff
@@ -38,6 +39,7 @@ class RecoInterface(ABC):
     rotate_vertex: bool
     refine_time: bool
     add_fallback_position: bool
+    use_pointing: bool
```

Comment:
I'm presuming that this is all `ABC` syntax magic :shrug: No change is needed if it works as designed, and I guess it does since it's an established pattern. Oh `ABC`...